client/reader/follow-button/index.jsx

```diff
@@ -16,9 +16,8 @@ const ReaderFollowButton = React.createClass( {
 	mixins: [ PureRenderMixin ],
 
 	propTypes: {
-		following: React.PropTypes.bool.isRequired,
```

Comment:
I sorta wonder if we can avoid the situation that led to this implementation pattern in the first place.